<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="FileMask">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Введите маску файлов (Ex: *.txt)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="DeleteSource">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалять исходные  файлы</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="FilesOutput">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Путь для результирующих файлов</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BrowseButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>135</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Обзор</string>
    </property>
   </widget>
   <widget class="QComboBox" name="DubleActions">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>171</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Перезапись</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Создание нового</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="HexKey">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>HEX key</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="SingleShot">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Разовый заупск</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Timer">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Запуск по таймеру</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="TimerInterval">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="StartStopButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>337</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Старт/Стоп</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>221</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="TextStatusLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Текстовый статус</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
